app/static/src/scss/record.scss

```diff
@@ -133,7 +133,7 @@
     }
 }
 
-.govuk-summary-list {
+.govuk-summary-list-record {
```

Comment:
was this name not following the bem or gds convention?

Reply:
spoke offline. this name is used elsewhere and so there was conflicting styles. instead extend the class for record page specifically